hknweb/tours/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.conf import settings
+from .models import DepTour
+from django.contrib.auth.models import User
+from django.forms.widgets import SelectDateWidget
+# from django.contrib.admin.widgets import AdminDateWidget
+
+import datetime
+
+
+class TourRequest(forms.ModelForm):
+    date = forms.DateField(widget=SelectDateWidget(), label='Desired Date')
+    desired_time = forms.TimeField(help_text='hh:mm 24-hour time', label='Desired Time')
```

Comment:
Since some students may be flying in from the East Coast or internationally, we should take into account time zones for scheduling a dept tour. Maybe specify that all of these times are in PST (-7:00 GMT) and/or put in some conversion like this https://www.timeanddate.com/worldclock/converter.html?iso=20200317T200000&p1=791. This particular website encodes listed time zones in the URL, so we can use that to our advantage!